server/konnectors/ameli.coffee

```diff
@@ -142,7 +146,7 @@ parsePage = (requiredFields, healthBills, data, next) ->
 
                     month = $($(this).find('.col-date .mois').get(0)).text()
```

Comment:
No, because there are 2 date locations : one with month and year concatenated (thus, the split is needed) and another one with month and day on their own field (.mois vs .jour). Hence my comment above :)